openshift_scalability/utils.py

```diff
@@ -34,12 +35,35 @@ def oc_command(args, globalvars):
 def login(user,passwd,master):
     return subprocess.check_output("oc login --insecure-skip-tls-verify=true -u " + user + " -p " + passwd + " " + master,shell=True)
 
+def get_route():
+    default_proj = subprocess.check_output("oc project default", shell=True)
+    localhost = subprocess.check_output("ip addr show eth0 | awk '/inet / {print $2;}' | cut -d/ -f1", shell=True).rstrip()
```

Comment:
@sjug : Are we always sure that we have to check eth0. I may be missing something obvious here, if not shouldn't we handle that scenario